Lib/sqlite3/dbapi2.py

```diff
@@ -20,9 +20,9 @@
 #    misrepresented as being the original software.
 # 3. This notice may not be removed or altered from any source distribution.
 
-import datetime
 import time
 import collections.abc
+import datetime
```

Comment:
One of these things doesn't belong ;-)
```suggestion
import collections.abc
import datetime
```

Reply:
A small import sort, which has been approved before :)
https://github.com/python/cpython/pull/129118

Reply:
Sorry, I should have been more explicit. I was referring to `import time` that's still present - I'm assuming you want to remove the module level import (as you did in https://github.com/python/cpython/pull/129118)

Reply:
It's particularly noticeable since the continued presence of `import time` makes this "sort the imports" stand out like a sore thumb due to not being sorted in the end.